toolchain/platform/defs.bzl

```diff
@@ -28,7 +28,7 @@ def declare_libc_aware_platforms():
                 "linux",
                 "linux",
                 suffix = "_{}".format(libc),
-                extra_constraints = ["@zig_sdk//libc:{}".format(libc)],
+                extra_constraints = ["//libc:{}".format(libc)],
             )
```

Comment:
The reason we can do this is because these starlark functions are always called from the generated repository, we do not need to use an external repository label here. We stay local to the {exec}-zig_sdk repository in these cases.